### PR TITLE
Document Mallet operator

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

File: docs/usage/playground.md [hunk 1]
```diff
@@ -90,6 +90,46 @@ var obj = {};
 if (!Object.prototype.hasOwnProperty.call(obj, "x")) obj.x = 2;
 ```
 
+### Mallet assignment operator
+
+The mallet assignment operator allows you to lazily initialize a variable.
+It checks whether the variable is falsey, if it is then the right hand value is set.
+
+While simplisticly equivalent to `a = a || b`, it will never set an already truthy value.
```
Comment (thejameskyle) [inline docs/usage/playground.md]: s/simplisticly/simplistically

File: docs/usage/playground.md [hunk 2]
```diff
@@ -90,6 +90,46 @@ var obj = {};
 if (!Object.prototype.hasOwnProperty.call(obj, "x")) obj.x = 2;
 ```
 
+### Mallet assignment operator
+
+The mallet assignment operator allows you to lazily initialize a variable.
+It checks whether the variable is falsey, if it is then the right hand value is set.
```
Comment (thejameskyle) [inline docs/usage/playground.md]: s/variable/value

**Comment by thejameskyle** [comment]
Thanks @jridgewell, just a couple of things and this is good to go.

File: docs/usage/playground.md (hunk 1 above)
Comment (jridgewell) [inline docs/usage/playground.md]: That's the last time I write prose in the Github editor...

**Comment by jridgewell** [comment]
Updated.

**Comment by thejameskyle** [comment]
Sweet thanks!

**Pull request merged**